docs/expression.md

```diff
@@ -5,8 +5,29 @@ For historical reasons, the parameters file for mRNA data is called
 `expression.params`.
 For the content of this file see [the HD data parameters](hd-params.md) and [the study-specific parameters](study-params.md).
 
-Input files
------------
+Annotation
+----------
+
+Parameters file name has to be `mrna_annotation.params`.
+`annotation.params` name is also supported as a legacy.
+
+**Attention:** The data file referred by `ANNOTATIONS_FILE` field  of the `mrna_annotation.params` file is expected to have a header (**first line is skipped**).
```

Comment:
You say the first line is skipped, but is used to identify the different columns right (instead of the order)?  I'm not sure about that now, should be made a little bit more clear.